src/Toolkit/V.php

```diff
@@ -248,6 +249,36 @@ public static function __callStatic(string $method, array $arguments): bool
                 $date['warning_count'] === 0;
     },
 
+    /**
+     * Compares the two provided values as dates in any format
+     * PHP can parse using the provided comparison operator
+     */
+    'dates' => function ($value, $test, $operator = '=='): bool {
+        $value = strtotime($value);
+        $test  = strtotime($test);
+
+        if (is_int($value) !== true || is_int($test) !== true) {
+            return false;
+        }
+
+        switch ($operator) {
+            case '==':
+                return $value == $test;
```

Comment:
Just thinking, @lukasbestle - we could use strict comparisons here, couldn't we?

Reply:
Definitely!

Reply:
The `$operator` values would still be unchanged for consistency with the collection filters, but the comparison itself can be strict.